pages/advanced-algorithms/available-algorithms/algo.mdx

```diff
@@ -0,0 +1,442 @@
+- `latitude_name: string, default = lat` - the name of the latitude property in nodes, which is needed for heuristic calculation. If unspecified,
+the default is used.
+- `longitude_name: string, default = lon` - the name of the longitude property in nodes, which is needed for heuristic calculation. If unspecified,
+the default is used.
+- `heuristic_name: string` - the name of the custom heuristic property. For example, if you don't want to use the provided heuristic,
+you can always define a custom heuristic property in your nodes, and use that. If this option is specified, even if the nodes are geospatial,
+the default heuristic is ignored.
+- `whitelisted_labels: List[string]` - a list of whitelisted labels. If this list is not empty, the algorithm will search for the path only through the node
+with specified labels. Example: `whitelisted_labels: ["City", "Village"]` means only nodes labelled `City` or `Village` will be considered.
+- `blacklisted_labels: List[string]` - a list of blacklisted labels. If this list is not empty, the algorithm will not search for the path through nodes with
+specified labels. Example: `blacklisted_labels: ["City", "Village"]` means nodes labelled `City` or `Village` will NOT be considered.
+- `relationships_filter: List[string]` - a list used for filtering relationships. If the list is empty, the algorithm will search the path
+through all possible relationships. If the list isn't empty, the algorithm will only search the path through the provided relationships.
+There are three possible ways to define a relationship inside the filter. Firstly: `<REL_NAME>`, means only incoming relationships with the
+provided name will be considered by the algorithm. `REL_NAME>` means only outgoing relationships with the provided name will be considered
+by the algorithm. Lastly, `REL_NAME` means that bout outgoing and incoming relationships with the provided name will be considered
+by the algorithm.
+- `duration: bool, default = false` - if `true`, means that the distance and heuristic are of a `Duration` data type. NOTE: If they are not created
+as a `Duration` data type, and the `duration` config is specified as `true`, an exception is thrown. Also, if this config is `true`, the `weight`
+output will be returned as `int`, representing the duration in microseconds.
+- `unweighted: bool, default = false` - if `true`, the algorithm will run unweighted, using only heuristics, not distances between nodes.
+This config will still return the path weight, but it will count each distance the same, as 10.
+- `epsilon: double, default = 1.0` - if the user has provided an admissible heuristic, or if they use the default heuristic,
+they can choose to use the Bounded relaxation property of A *, and multiply the heuristic by `epsilon` value. This will result in a faster algorithm, with a solution
+at most `epsilon` times worse than the base algorithm. See [more](https://en.wikipedia.org/wiki/A*_search_algorithm#Bounded_relaxation).
```

Comment:
```suggestion
- `distance_prop: string, default = distance` - The name of the relationships' distance property. 
- `latitude_name: string, default = lat` - The name of the nodes' latitude property,  needed for the heuristic calculation.
- `longitude_name: string, default = lon` - The name of the nodes' longitude property, needed for the heuristic calculation.
- `heuristic_name: string` - The name of the custom heuristic property. For example, if you don't want to use the provided heuristic,
you can always define and use a custom heuristic node property. If the `heuristic_name` is set to a certain value, the default heuristic is ignored.
- `whitelisted_labels: List[string]` - A list of whitelisted labels through which the algorithm will search for the path. For example, by setting `whitelisted_labels: ["City", "Village"]` the algorithm will consider only nodes labeled `City` or `Village`.
- `blacklisted_labels: List[string]` - A list of blacklisted labels, which will be ignored by the algorithm when searching for a path.  For example, by setting `blacklisted_labels: ["City", "Village"]` the algorithm will ignore nodes labeled `City` or `Village`.
- `relationships_filter: List[string]` - A list used for filtering relationships. If the list is empty, the algorithm will search for the path
through all possible relationships. If the list isn't empty, the algorithm will search for the path only through the listed relationships.
There are three possible ways to define a relationship inside the filter: 
- `<REL_NAME>` - Only the incoming relationships with the defined name will be considered by the algorithm.
- `REL_NAME>`- Only the outgoing relationships with the defined name will be considered by the algorithm. 
- `REL_NAME` - Both outgoing and incoming relationships with the defined name will be considered
by the algorithm.
- `duration: bool, default = false` - If `true`, means that the distance and heuristic properties are of a `Duration` data type. If they are not, but the `duration` config is specified as `true`, an exception is thrown. Also, if this config is `true`, the `weight` output will be returned as `int`, representing the duration in microseconds.
- `unweighted: bool, default = false` - if `true`, the algorithm will ignore the weights, using only heuristics, not distances between nodes. The path weight will still be returned, but all distances will be of the same value 10.
- `epsilon: double, default = 1.0` - If an admissible heuristic has been provided, or if the default heuristic is used,
you can choose to use the [Bounded relaxation](https://en.wikipedia.org/wiki/A*_search_algorithm#Bounded_relaxation) property of A *, and multiply the heuristic by `epsilon` value. This will result in a faster algorithm, with a solution at most `epsilon` times worse than the base algorithm.
```